test/utils/shallowEquality.js

```diff
@@ -0,0 +1,68 @@
+import expect from 'expect';
+import shallowEqualScalar from '../../src/utils/shallowEqualScalar';
+import shallowEqual from '../../src/utils/shallowEqual';
+
+describe('Utils', () => {
+  describe('shallowEqualScalar', () => {
+    it('should compare the given arguments shallowly', () => {
+
+      expect(shallowEqualScalar(null, undefined)).toBe(false);
```

Comment:
Let's only test things whose `typeof` is `object`. This includes `null`, but not `undefined`.